gnovm/stdlibs/testing/testing.gno

```diff
@@ -363,3 +413,79 @@ func tRunner(t *T, fn testingFunc, verbose bool) {
 
 	fn(t)
 }
+
+func testSetContext(
+	isOrigin bool,
+	originCaller string,
+	originPkgAddress string,
+	currRealmAddr string, currRealmPkgPath string,
+	origSendDenoms []string, origSendAmounts []int64,
+	origSpendDenoms []string, origSpendAmounts []int64,
+	chainID string,
+	height int64,
+	timeUnix int64, timeNano int64,
+)
+
+func testIssueCoins(addr string, denom []string, amt []int64)
+
+func TestSetContext(ctx Context) {
```

Comment:
Second thought, maybe let's just make this `SetContext` for consistency with others and also because `testing.SetContext` rolls off better